frontend/src/components/ImageComponent.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
This might be a bit unnecessary. It is essentially doing nothing but changing the tag from `<img></img>` to `<ImageComponent><ImageComponent/>`